tests/func/tasks/admin/clean/user_project_role_assignment.yml

```diff
@@ -0,0 +1,10 @@
+- name: remove osm_user_project_role_assignment
+  openstack.cloud.role_assignment:
+    auth: "{{ item['auth'] }}"
+    name: "{{ item['user_project_role_assignment'] }}"
```

Comment:
Instead of `name` we'll have `user`, `role` and `project` here.

Reply:
`project` will be `osm_project` https://github.com/os-migrate/os-migrate/blob/1dd1e1c95759d386fadb42fcaf0b2c64ec97a456/tests/func/tasks/admin/seed/project.yml#L4

`user` will be `osm_user` https://github.com/os-migrate/os-migrate/blob/1dd1e1c95759d386fadb42fcaf0b2c64ec97a456/tests/func/tasks/admin/seed/user.yml#L4

`role` i'm not sure but perhaps `_member_` or `member`, or if those don't work or are already assigned, we could try `admin`, or if that causes issues, we could see if we can create our own role for testing purposes in the `seed` tasks